### PR TITLE
Migrating to Gradle

### PR DESCRIPTION
**Pull request by YuliaShvets**

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.5.0"
+}
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
```
Comment (ajax-gnatiuk-s) [inline settings.gradle.kts]: let's remove this comment

**Review by ajax-gnatiuk-s — CHANGES_REQUESTED** [review]
Few more actions are required:
- commit `gradew` and `gradlew.bat` files 
- commit all files from dir `gradle/wrapper`                     

look at PR to see what files must be present in your project https://github.com/OlehSarapuk/ajax-onboarding-project/pull/5/files

File: settings.gradle.kts (hunk 1 above)
Comment (YuliaShvets) [reply settings.gradle.kts]: ✅

File: .gitignore [hunk 2]
```diff
@@ -7,19 +7,14 @@ target
 
 # BlueJ files
 *.ctxt
-.gradle
```
Comment (ajax-gnatiuk-s) [inline .gitignore:10]: this dir should be ignored. Please rollback the row

File: .gitignore [hunk 3]
```diff
@@ -7,19 +7,14 @@ target
...
-gradlew.bat
 build
-gradlew
-gradle
 # Mobile Tools for Java (J2ME)
 .mtj.tmp/
 
 
 # Package Files #
-*.jar
```
Comment (ajax-gnatiuk-s) [inline .gitignore:22]: You should leave this in your .gitignore. 

gradle wrapper jar is the only file we are interested to commit. So, just rollback this row

**Review by ajax-gnatiuk-s — CHANGES_REQUESTED** [review]
LGTM. Just fix your gitignore and let's merge

File: .gitignore (hunk 2 above)
Comment (YuliaShvets) [reply .gitignore:10]: ✅

File: .gitignore (hunk 3 above)
Comment (YuliaShvets) [reply .gitignore:22]: ✅

**Pull request merged**